Servers/vm-hardware-version-upgrade.md

```diff
@@ -0,0 +1,55 @@
+{{{
+	"title": "Virtual Machine Hardware Version Upgrade",
+	"date": "05-24-2018",
+	"author": "Matt Schwabenbauer",
+	"attachments": [],
+	"related-products" : [],
+	"contentIsHTML": false,
+	"sticky": true
+}}}
+
+### Description
+On May 23, 2018, CenturyLink Cloud upgraded the Virtual Machine (VM) hardware version for all VMs in each of our public cloud service locations. This upgrade was released in order to apply guest-level protection from the Meltdown and Spectre security vulnerabilities.
+
+### Applying the update
+
+In order for the update to take effect, all VMs in each CenturyLink Cloud service location will need to be issued a shut down command followed by a power on command. Customers may issue the "Shut Down" server power operation from the Control Portal or API, followed by the "On" server power operation to apply the update.
+
+NOTE: VMs running Windows Operating Systems must upgrade their VMware Tools before following the subsequent steps to complete this update. Instructions on updating the VMware Tools for VMs in CenturyLink Cloud can be [found here](https://www.ctl.io/knowledge-base/servers/self-service-updates-of-vmware-tools/)
```

Comment:
should be relative links to the kb .md files not the URL itself

Reply:
Why? Shouldn’t it work either way?